src/forgeAPI/java/net/neoforged/api/distmarker/Dist.java

```diff
@@ -29,6 +29,48 @@
  *     <li>{@link #DEDICATED_SERVER} is the <em>dedicated server</em> distribution,
  *     it contains a server, which can simulate the world and communicates via network.</li>
  * </ul>
+ * <p>
+ * When working with Dist-specific code, it is important to guard invocations such that
```

Comment:
I think the term "Dist-specific code" needs more clarification, or at least the term "Dist" being expanded fully.